queue_job/job.py

```diff
@@ -1,32 +1,32 @@
-# Copyright 2013-2016 Camptocamp
+# Copyright 2013-2020 Camptocamp
 # License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html)
 
-import inspect
 import functools
 import hashlib
+import inspect
 import logging
-import uuid
+import os
 import sys
+import uuid
 from datetime import datetime, timedelta
 
 import odoo
 
-from .exception import (NoSuchJobError,
-                        FailedJobError,
-                        RetryableJobError)
+from .exception import FailedJobError, NoSuchJobError, RetryableJobError
```

Comment:
What do you think about undoing the changes from python-black in this PR? It would make it a lot easier to review.